packages/azure/data_stream/firewall_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -194,6 +194,12 @@ processors:
         - "^%{DATA:azure.firewall.proto} request from %{IPORHOST:source.address}:%{NUMBER:source.port:long} to %{IPORHOST:destination.address}:%{NUMBER:destination.port:long}. Url: %{HOSTNAME:url.original}. Action: %{DATA:azure.firewall.action}. ThreatIntel: %{DATA:rule.name}$" 
       if: ctx?.json?.operationName == 'AzureFirewallNetworkRuleLog' || ctx?.json?.operationName == 'AzureFirewallNatRuleLog'
 
+  - grok:
+      field: json.properties.msg
+      patterns:
+        - "^%{DATA:azure.firewall.proto} request from %{IPORHOST:source.address}:%{NUMBER:source.port:long} to %{IPORHOST:destination.address}:%{NUMBER:destination.port:long}. Action: %{DATA:azure.firewall.action}. ThreatIntel: %{DATA:rule.name}$" 
+      if: ctx?.json?.operationName == 'AzureFirewallThreatIntelLog'
```

```suggestion
      if: ctx.json?.operationName == 'AzureFirewallThreatIntelLog'
```
`ctx` is always non-null.